### PR TITLE
Feat: Video Card Component On Live

### PR DESCRIPTION
**Pull request by Rishi05051997**
Peer Members, please review the video card component 
Video-card

### CODE REVIEW

File: src/components/VideoCard/videocard.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.container-card {
+    background-color: var(--background-color-light) !important;
+    box-shadow: 4px 4px 8px -2px var(--primary-light-color) !important;
+}
+
+.container-card:hover{
+    transform: scale(1.05);
+    box-shadow: 0 10px 20px rgba(0,0,0,.12), 0 4px 8px rgba(0,0,0,.06) !important;
+}
```
Comment (Mohasin-Haque) [inline src/components/VideoCard/videocard.css:9]: Why !important ?? Can you fix it..

**Review by Mohasin-Haque — COMMENTED** [review]
Your code looks good. You can work on suggetions

File: src/components/VideoCard/VideoCard.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    const singleVideoNavigator = () => {
+        navigate(`/single-video/${_id}`)
+    }
+
+    const isPlaylistPresent = playlists.find(({ _id }) => _id === playlistId)
+
+    const removeHandler = (dispatch, video, token, action, playlistId) => {
+        switch (action) {
+            case "REMOVE-FROM-LIKE":
+                disLikeVideoBasedBasedOnVideoId(dispatch, video, token)
```
Comment (vishalg8454) [inline src/components/VideoCard/VideoCard.jsx]: `disLikeVideoBasedBasedOnVideoId` - possibly a typo.

**Review by vishalg8454 — APPROVED** [review]
All looks good. As @Mohasin-Haque commented, try to remove !important if possible.

File: src/components/VideoCard/VideoCard.jsx (hunk 2 above)
Comment (Rishi05051997) [reply src/components/VideoCard/VideoCard.jsx]: Yes Changing sorry my bad

File: src/components/VideoCard/videocard.css (hunk 1 above)
Comment (Rishi05051997) [reply src/components/VideoCard/videocard.css:9]: I'm overriding component libraries classes

**Pull request merged**